lib/roda/plugins/endpoint.rb

```diff
@@ -25,16 +25,18 @@ module RodaPlugins
     #     end
     #   end
     module Endpoint
-      # @param app [Any]
-      def self.configure(app)
+      # @param app [Roda] the Roda application
+      # @param name [String] override the class name of the Roda application
+      def self.configure(app, name: nil)
+        app.opts[:endpoint_class_name] = name || app.name
```

Comment:
Тут случайно не надо вот так поправить?

```suggestion
        app.opts[:endpoint_class_name] = name || app.class.name
```

Reply:
неа, app это уже class